src/myr/reference.js

```diff
@@ -226,6 +226,13 @@ let transformations = [
         description: <span>The resetCursor function resets the properties of the cursor to their defaults. This includes cursor attributes pertaining to Transformations and Animations.</span>,
         example: "resetCursor"
     },
+    {
+        name: "resetTransformationCursor",
+        parameters: [],
+        description: <span>The resetTransformationCursor function resets the transformation properties of the cursor to their defaults.</span>,
+        example: "resetTransformationCursor"
```

Comment:
The example property should only be added after Elena or I add the example to the database

Please also work with @elenaizotova on the wording of these.